src/utils/Api.js

```diff
@@ -1,4 +1,19 @@
 
-export default class SHEETS_API {
+const apiUrl = 'http://localhost:8003/graphql';
```

Comment:
should go into .env.development and have a corresponding value in .env.development of dashboard-spa